src/components/Pages/index.js

```diff
@@ -41,10 +41,10 @@ class VmDetailPage extends React.Component {
   }
 
   render () {
-    let { match, vms, config } = this.props
+    let { match, vms, config, requestActive } = this.props
     if (vms.getIn(['vms', match.params.id])) {
       return (<VmDetail vm={vms.getIn(['vms', match.params.id])} config={config} />)
-    } else if (vms.get('loadInProgress')) {
+    } else if (requestActive) {
```

Comment:
So on a VM view page, click the Run (or Stop) button.  The **requestActive** will go true for the duration of the REST request.  Will the page then render a null component and leave the user blank until the REST request is complete?

On a good note, the next PR I'm posting (to deal with #574) will remove this logic all together in this location.

Reply:
The page will be rendered correctly (verified) since condition `vms.getIn(['vms', match.params.id])` at line 45 evaluates to `true`.